### PR TITLE
[RA-TES-6311] Drill down to detail test run

### PR DESCRIPTION
**Pull request by quanvo0213**
### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/katalon/CustomFilter/FilterExtension.tsx [hunk 1]
```diff
@@ -146,11 +146,15 @@ function FilterExtension(props: FilterExtensionProps) {
...
+                {item.element}
+              </div>
             ))}
           {addFilter.length !== 0 &&
             addFilter.map((item, index) => (
-              <div key={index}>{item.element}</div>
+              <div id={item.name} key={index}>
+                {item.element}
+              </div>
             ))}
```
Comment (lequangtrai) [inline superset-frontend/src/katalon/CustomFilter/FilterExtension.tsx:158]: check merge main this. I think that we update the logic here
@triquanminh

**Review by lequangtrai — CHANGES_REQUESTED** [review]

File: superset-frontend/src/katalon/CustomFilter/FilterExtension.tsx (hunk 1 above)
Comment (quanvo0213) [reply superset-frontend/src/katalon/CustomFilter/FilterExtension.tsx:158]: done

**Review by huynguyen2908 — APPROVED** [review]

File: superset-frontend/src/utils/getIsKatalonEmbeddedDashboard.ts [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+  // @ts-ignore
+  getUrlParam(KATALON_URL_PARAMS.isKatalonEmbeddedMode);
+
+export const getKatalonProjectId = () =>
+  // @ts-ignore
+  getUrlParam(KATALON_URL_PARAMS.projectId);
+
+export const getKatalonIsMetric = () =>
+  // @ts-ignore
+  getUrlParam(KATALON_URL_PARAMS.isMetric);
```
Comment (triquanminh) [inline superset-frontend/src/utils/getIsKatalonEmbeddedDashboard.ts]: Given that this utility file handles various URL parameters, I think we should update the file name to something such as `getKatalonUrlParams.ts`

**Review by lequangtrai — APPROVED** [review]

**Review by triquanminh — APPROVED** [review]

File: superset-frontend/src/utils/getIsKatalonEmbeddedDashboard.ts (hunk 2 above)
Comment (quanvo0213) [reply superset-frontend/src/utils/getIsKatalonEmbeddedDashboard.ts]: solved

**Pull request merged**